### ISSUE TITLE
Database container not running on pantheon mysql import

### ISSUE THREAD

**Issue by soniktrooth** [kalabox/kalabox#789]
So, I have this friend...oh never mind—I ran `kbox create pantheon` and chose greenbiz. Lots and lots of things happened, I entered teh Matrix and then this happened:

```
Error dispatching event listeners [post-create-app]: Error while running command "import-mysql,localhost,,3306,/src/config/terminus//greenbiz_sprint_2015-12-04T19-00-00_UTC_database.sql.gz" in container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running

VError: Error while running command "import-mysql,localhost,,3306,/src/config/terminus//greenbiz_sprint_2015-12-04T19-00-00_UTC_database.sql.gz" in container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running

    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:624:13
From previous event:
    at exec (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:623:11)
    at query (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:665:12)
    at Object.queryData (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:675:12)
From previous event:
    at /usr/local/bin/lib/engine.js.jx:682:29
From previous event:
    at Object.exports.queryData (/usr/local/bin/lib/engine.js.jx:681:4)
    at /usr/local/bin/greenbiz/plugins/kalabox-plugin-pantheon/lib/pull.js:163:21
From previous event:
    at Object.pullDB (/usr/local/bin/greenbiz/plugins/kalabox-plugin-pantheon/lib/pull.js:160:6)
    at /usr/local/bin/greenbiz/plugins/kalabox-plugin-pantheon/lib/tasks.js:560:23
From previous event:
    at Object.fn (/usr/local/bin/greenbiz/plugins/kalabox-plugin-pantheon/lib/tasks.js:559:8)
    at /usr/local/bin/lib/core/events.js.jx:178:14
    at /usr/local/bin/lib/core/events.js.jx:177:24
From previous event:
    at /usr/local/bin/lib/core/events.js.jx:161:23
From previous event:
    at EventEngine._dispatch (/usr/local/bin/lib/core/events.js.jx:158:18)
    at /usr/local/bin/lib/core/events.js.jx:137:19
From previous event:
    at EventEngine.emit (/usr/local/bin/lib/core/events.js.jx:129:21)
From previous event:
    at Object.exports.emit (/usr/local/bin/lib/core/events.js.jx:261:4)
    at Promise.try.then.catch.then.then.then.then.then.then.then.then.then.then.sysRoot (/usr/local/bin/lib/create.js.jx:421:26)
    at process._tickCallback (node.js:803:1)
From previous event:
    at createApp (/usr/local/bin/lib/create.js.jx:419:6)
    at PromptUI.completed (/usr/local/bin/lib/create.js.jx:534:18)
    at PromptUI.onCompletion (/usr/local/bin/node_modules/inquirer/lib/ui/prompt.js.jx:68:10)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.completed (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1792:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1729:14)
    at AnonymousObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1729:14)
    at InnerObserver.onCompleted (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:3424:65)
    at InnerObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1729:14)
    at AutoDetachObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1729:14)
    at InnerObserver.onCompleted (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:3424:65)
    at InnerObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1729:14)
```

I'm now stuck inside the Matrix and can't get out. pls help

**Comment by pirog** [comment]
@soniktrooth have you gone into that app and tried to run `kbox pull` again?

**Comment by soniktrooth** [comment]
@pirog I ran `kbox greenbiz pull` and I got this error:

```
Error while running command "import-mysql,localhost,,3306,/src/config/terminus//greenbiz_sprint_2015-12-06T19-00-00_UTC_database.sql.gz" in container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running

Error: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running

    at /usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:217:17
    at getCause (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:245:7)
    at Modem.buildPayload (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:216:5)
    at null.<anonymous> (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:192:14)
    at emit (events.js:98:14)
    at _stream_readable.js:889:8
    at process._tickCallback (node.js:803:1)
```

I'm in a glass case of emotion

**Comment by labboy0276** [comment]
Is this a multidev?  Looks like it.

On Mon, Dec 7, 2015 at 1:04 PM, Josh <notifications@github.com> wrote:

> @pirog <https://github.com/pirog> I ran kbox greenbiz pull and I got this
> error:
>
> Error while running command "import-mysql,localhost,,3306,/src/config/terminus//greenbiz_sprint_2015-12-06T19-00-00_UTC_database.sql.gz" in container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running
>
> Error: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running
>
>     at /usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:217:17
>     at getCause (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:245:7)
>     at Modem.buildPayload (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:216:5)
>     at null.<anonymous> (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:192:14)
>     at emit (events.js:98:14)
>     at _stream_readable.js:889:8
>     at process._tickCallback (node.js:803:1)
>
> I'm in a glass case of emotion
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/789#issuecomment-162609830>.
>



-- 
*John Ouellet*
*Sr Developer & Support Manager*
415-470-3076
www.kalamuna.com

Stay connected with Kalamuna on: Twitter
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Ftwitter.com%2Fkalamuna&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>
, Linkedin
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.linkedin.com%2Fcompany%2F2584606&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>,
and Facebook
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.facebook.com%2Fkalamunatech&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>

**Comment by soniktrooth** [comment]
I don't think so—would that make a difference?

**Comment by labboy0276** [comment]
not sure, sometimes, but /greenbiz_sprint is in the terminus command, why I
asked

On Mon, Dec 7, 2015 at 1:12 PM, Josh <notifications@github.com> wrote:

> I don't think so—would that make a difference?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/789#issuecomment-162611645>.
>



-- 
*John Ouellet*
*Sr Developer & Support Manager*
415-470-3076
www.kalamuna.com

Stay connected with Kalamuna on: Twitter
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Ftwitter.com%2Fkalamuna&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>
, Linkedin
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.linkedin.com%2Fcompany%2F2584606&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>,
and Facebook
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.facebook.com%2Fkalamunatech&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>

**Comment by soniktrooth** [comment]
Ok, I lied—it is a multidev

**Comment by soniktrooth** [comment]
I tried pulling the live env instead of sprint and got:

```
Error while running command "import-mysql,localhost,,3306,/src/config/terminus//greenbiz_live_2015-12-07T08-00-00_UTC_database.sql.gz" in container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running

Error: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running

    at /usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:217:17
    at getCause (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:245:7)
    at Modem.buildPayload (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:216:5)
    at null.<anonymous> (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:192:14)
    at emit (events.js:98:14)
    at _stream_readable.js:889:8
    at process._tickCallback (node.js:803:1)
```

**Comment by pirog** [comment]
Looks like the same issue. I'll try to replicate this.

On Mon, Dec 7, 2015 at 10:34 AM, Josh <notifications@github.com> wrote:

> I tried pulling the live env instead of sprint and got:
>
> Error while running command "import-mysql,localhost,,3306,/src/config/terminus//greenbiz_live_2015-12-07T08-00-00_UTC_database.sql.gz" in container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running
>
> Error: HTTP code is 500 which indicates error: server error - Container 13df478fd1107a41e09ef847bf3d3619e0d1cfdda49ad16cd915d017fe0b2d40 is not running
>
>     at /usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:217:17
>     at getCause (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:245:7)
>     at Modem.buildPayload (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:216:5)
>     at null.<anonymous> (/usr/local/bin/node_modules/kalabox-engine-docker/node_modules/dockerode/node_modules/docker-modem/lib/modem.js.jx:192:14)
>     at emit (events.js:98:14)
>     at _stream_readable.js:889:8
>     at process._tickCallback (node.js:803:1)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/789#issuecomment-162616931>.
>



-- 

Mike

-- 
Mike Pirog
Co-Founder, Products
http://kalamuna.com

Stay connected with Kalamuna on: Twitter
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Ftwitter.com%2Fkalamuna&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>
, Linkedin
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.linkedin.com%2Fcompany%2F2584606&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>,
and Facebook
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.facebook.com%2Fkalamunatech&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>

**Comment by soniktrooth** [comment]
@pirog you Sir, are a gentleman and a scholar.

**Comment by katypool** [comment]
Hey, this is happening to me too!
Then when I go into the app and run kbox pull (pulling db and files from dev), I get this error:

Failed after 5 retries. {"max":5,"backoff":500}: Cannot call method 'split' of undefined
TypeError: Cannot call method 'split' of undefined
    at /Users/katy/Desktop/apps/kalablog/plugins/kalabox-plugin-pantheon/lib/terminus.js:357:35
From previous event:
    at /Users/katy/Desktop/apps/kalablog/plugins/kalabox-plugin-pantheon/lib/terminus.js:345:11
    at /usr/local/bin/lib/promise.js.jx:44:16
From previous event:
    at rec (/usr/local/bin/lib/promise.js.jx:43:25)
    at /usr/local/bin/lib/promise.js.jx:55:20
    at process._tickCallback (node.js:803:1)

Pulling from any other environment (live or test) gets the same error Josh had. This site also has multidev environments, but I'm just creating the app from dev.
Running kalabox 10.6

**Comment by pirog** [comment]
I think this should be resolved in v0.11
